src/viam/ur/module/ur_arm.cpp

```diff
@@ -552,10 +565,15 @@ void URArm::move(std::vector<Eigen::VectorXd> waypoints, std::chrono::millisecon
 
     for (size_t i = 0; i < segments.size() - 1; i++) {
         const auto start = boost::numeric_cast<decltype(waypoints)::difference_type>(segments[i]);
+
         const auto end = boost::numeric_cast<decltype(waypoints)::difference_type>(segments[i + 1] + 1);
+
         const std::list<Eigen::VectorXd> positions_subset(waypoints.begin() + start, waypoints.begin() + end);
+
         const Trajectory trajectory(Path(positions_subset, 0.1), max_velocity, max_acceleration);
+
         trajectory.outputPhasePlaneTrajectory();
+
```

Comment:
Can we collapse this back together? I have changes here that will conflict annoyingly otherwise.